core/geometry/src/test/polyface/Polyface.test.ts

```diff
@@ -127,394 +125,299 @@ function exercisePolyface(ck: Checker, polyface: Polyface, longEdgeIsHidden: boo
-      Point3d.create(ax0 * f, ay / f, 0),    // area matches
-      Point3d.create(ax0 * 2, ay, 0),    // same orientation, different radii.
-      Point3d.create(ax0, ay * 0.9, 0),    // same orientation, different radii.
-      Point3d.create(ay, ax0, 0),    // rotate 90 degrees, same radii
-    ]) {
-      const builder = PolyfaceBuilder.create();
-      const bx = corner.x;
-      const by = corner.y;
-      const expectSameMoments = corner.z === 1;
-      // positive polygon
-      builder.addPolygon([Point3d.create(-ax0, -ay, zTop), Point3d.create(ax0, -ay, zTop), Point3d.create(ax0, ay, zTop), Point3d.create(-ax0, ay, zTop)]);
-      builder.addPolygon([Point3d.create(-bx, -by, zBottom), Point3d.create(-bx, by, zBottom), Point3d.create(bx, by, zBottom), Point3d.create(bx, -by, zBottom)]);
-      const polyface = builder.claimPolyface();
-      const volumeData = PolyfaceQuery.sumVolumeBetweenFacetsAndPlane(polyface, zPlane);
-      ck.testBoolean(expectSameMoments, MomentData.areEquivalentPrincipalAxes(volumeData.positiveProjectedFacetAreaMoments, volumeData.negativeProjectedFacetAreaMoments), "Expect mismatched moments");
-    }
-    expect(ck.getNumErrors()).equals(0);
-  });
+it("Polyface.RaggedBoxVolume", () => {
+  const ck = new Checker();
+  const builder = PolyfaceBuilder.create();
+  const a = 2; const b = 3; const c = 4;
+  const expectedVolume = a * b * c;
+  const expectedAreaZX = a * c;
+  const xzPlane = Plane3dByOriginAndUnitNormal.createZXPlane();
+  const openBox = Box.createRange(Range3d.createXYZXYZ(0, 0, 0, a, b, c), false);
+  builder.addBox(openBox!);
+  const polyface = builder.claimPolyface();
+  // the box is open top and bottom !!
+  const volumeZX = PolyfaceQuery.sumVolumeBetweenFacetsAndPlane(polyface, xzPlane);
+  ck.testDefined(volumeZX.positiveProjectedFacetAreaMoments);
+  ck.testDefined(volumeZX.negativeProjectedFacetAreaMoments);
+  if (volumeZX.positiveProjectedFacetAreaMoments && volumeZX.negativeProjectedFacetAreaMoments) {
+    ck.testCoordinate(expectedAreaZX, volumeZX.positiveProjectedFacetAreaMoments.quantitySum);
+    ck.testCoordinate(expectedAreaZX, volumeZX.negativeProjectedFacetAreaMoments.quantitySum);
+    ck.testCoordinate(expectedVolume, volumeZX.volume);
+    ck.testCentroidAndRadii(volumeZX.positiveProjectedFacetAreaMoments, volumeZX.negativeProjectedFacetAreaMoments, "open box ragged moments");
+  }
+  // In other planes, the missing facets are NOT perpendicular, and we expect to detect the mismatched projections in the moments.
+  const planeB = Plane3dByOriginAndUnitNormal.createXYZUVW(0, 0, 0, 1, 2, 3)!;
+  const volumeB = PolyfaceQuery.sumVolumeBetweenFacetsAndPlane(polyface, planeB);
+  ck.testFalse(MomentData.areEquivalentPrincipalAxes(volumeB.positiveProjectedFacetAreaMoments, volumeB.negativeProjectedFacetAreaMoments), "Expect mismatched moments");
+  expect(ck.getNumErrors()).equals(0);
+});
 
-  it("Polyface.IntersectLocalRangeBoxes", () => {
```

Comment:
As mentioned, "Polyface.IntersectLocalRangeBoxes" is moved to "ClipUtilities.test.ts"